IOS_CREDENTIALS_EAS.md

```diff
@@ -0,0 +1,138 @@
+# iOS Credentials Guide: OneSignal + EAS
+## Overview
+To distribute your iOS application, you must set up all the credentials required by Apple. The 3 primary iOS credentials are:
+* Distribution certificate
+* Provisioning profiles
+* Push notification certificate
+
+If you are using EAS, you may be letting Expo handle all your credentials. This is fine for your distribution and provisioning credentials. However, the push notification certificate *must* be handled separately since you are using OneSignal.
+
+As of 3/22/2022, OneSignal does not support p8 files and EAS does not support p12 files which results in a particlar issue when cloud building through EAS while using the Expo managed credentials service. Since the EAS CLI only prompts for p8 files at the time of writing, the only way to complete the cloud build is to input "n" (no) when prompted for setting up push notifications through the CLI. However, this leads to the identifier not having the push capability required to set up push successfully. Until EAS adds more granular control over credentials or OneSignal adds p8 support or Expo adds p12 support, developers will have no option except to use local credentials only.
```

Comment:
We could instruct them to say yes on the p8 so the push capability is added to the provisioning profile. Of course they won't be able to use it with OneSignal to send notifications but they can simply use the [OneSignal provisionator tool](https://onesignal.com/provisionator) right after to generate a .p12 file.

Reply:
The CLI actually asks for a path to the p8 file before proceeding. This pushes it up to EAS. I suppose your solution might work, but we may have to provide a dummy p8 -- which could lead to some confusion. What do you think?

Reply:
O I see, my confusion was I though the CLI was generating it for the app developer. Marking this is resolved as what you have make sense now.